lib/output.js

```diff
@@ -168,6 +168,9 @@ const png = function png (options) {
  * Use these WebP options for output image.
  * @param {Object} [options] - output options
  * @param {Number} [options.quality=80] - quality, integer 1-100
+ * @param {Number} [options.alphaQuality=100] - quality of alpha layer, integer 0-100
+ * @param {Boolean} [options.lossless=false] - use lossless compression mode
+ * @param {Boolean} [options.nearLossless=false] - use near_lossless compression mode
```

Comment:
`nearLossless` has to be used in conjunction with Q. `nearLossless` with Q=100 is equivalent to  `lossless` . But with other quality settings, it will lead to a smaller file size than `lossless`